platforms/android/library/src/main/java/io/element/android/wysiwyg/EditorStyledTextView.kt

```diff
@@ -55,22 +65,29 @@ open class EditorStyledTextView : AppCompatTextView {
         codeBlockBgHelper.clearCachedPositions()
     }
 
+    /**
+     * Sets up the [HtmlConverter] used to translate HTML to Spanned text.
+     * @param styleConfig The styles to use for the generated spans.
+     * @param mentionDisplayHandler Used to decide how to display any mentions found in the HTML text.
+     */
+    fun setupHtmlConverter(styleConfig: StyleConfig, mentionDisplayHandler: MentionDisplayHandler?) {
```

Comment:
Same comment as above, please can we rename the public API to something more friendly?